internal/controllers/hostedclusters/hostedcluster_controller.go

```diff
@@ -130,7 +130,7 @@ func (c *HostedClusterController) desiredPackage(cluster *v1beta1.HostedCluster)
 		config["affinity"] = c.remotePhaseAffinity
 	}
 	if c.remotePhaseTollerations != nil {
-		config["tollerations"] = c.remotePhaseTollerations
```

Comment:
should we also fix the field name to avoid confusion? s/remotePhaseTollerations/remotePhaseTolerations/

Reply:
:man_facepalming: Thanks!
Updated.